reame.txt

```diff
@@ -0,0 +1,5 @@
+Głównym celem tego projektu było zapoznanie się ze rodzajami list, ich implementacją oraz ręcznym i automatycznym scrollowaniem.
+Treść strony została pobrana z kursu od Mirosława Zelenta.
```

Comment:
Use Markdowntags within this file!
Read more and make familiar with Markdown syntax as well.